pkg/Microsoft.Private.CoreFx.NETCoreApp/Microsoft.Private.CoreFx.NETCoreApp.props

```diff
@@ -41,7 +42,9 @@
     <_project Include="@(BuildRID)">
       <Platform Condition="'%(Platform)' == ''">amd64</Platform>
       <PackageTargetRuntime>%(Identity)</PackageTargetRuntime>
-      <AdditionalProperties>PackageTargetRuntime=%(Identity);Platform=%(Platform)</AdditionalProperties>
+      <RuntimeOS>$(RuntimeOS)</RuntimeOS>
```

Comment:
Per our discussion let's just set SkipBuildConfigurationImport in the pkgproj, then make sure that the builds passes ConfigurationGroup down.